cmd/thanos/sidecar.go

```diff
@@ -355,3 +409,8 @@ func (s *promMetadata) Timestamps() (mint int64, maxt int64) {
 
 	return s.mint, s.maxt
 }
+
+func (s *promMetadata) FetchPromVersion(logger log.Logger) (err error) {
```

Comment:
```suggestion
func (s *promMetadata) FetchPromVersion(logger log.Logger) error {
```

Reply:
As per https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

Reply:
We could remove this all together, as it's one line wrapper arround `_, err := promclient.PromVersion(logger, s.promURL)`

So I would prefer to use that directly.

Reply:
Done.